internal/web_server/web/handlers/dashboard_pages_handlers.go

```diff
@@ -148,3 +151,22 @@ func (dph *DashboardPagesHandler) userDetails(c echo.Context) error {
 func addMenuContext(c echo.Context, menuContext side_menu.SideMenuItem) {
 	utils.AddToContext(c, side_menu.MENU_CONTEXT_KEY, menuContext)
 }
+
+// Gets the usernames administration page and returns it.
+func (dph *DashboardPagesHandler) getUserNameAdministration(c echo.Context) error {
+	aPage, err := strconv.Atoi(c.QueryParam("apage"))
+	if err != nil { // If the page number is not a number, set it to 1.
+		aPage = 1
+	}
+	nPage, err := strconv.Atoi(c.QueryParam("npage"))
+	if err != nil { // If the page number is not a number, set it to 1.
+		nPage = 1
+	}
```

Comment:
Don't know if we decided on a convention, but would it not be easier for readability and understanding to have aPage and nPage be written out in full, e.g. adjectivePage

Reply:
changed it to adjPage and nounPage